### PR TITLE
Some messages mark as traslatable

### PR DESCRIPTION
**Pull request by kaakaa**
#### Summary
Some messages in the following page aren't marked as translatable.
* Purchase modal

スクリーンショット 2020-12-30 23 18 46

* Next Step Tips for channel section (This was changed to an untranslatable message in [this commit](https://github.com/mattermost/mattermost-webapp/pull/6514/commits/9b51fde626b9c38cd8f08fcb968d04657f02f496). I think this changes should be reverted.)

スクリーンショット 2020-12-30 23 09 08

I couldn't see if these changes fix issues in browser, because I couldn't open these pages for MM cloud in my local environment.

#### Ticket Link
N/A

#### Related Pull Requests
N/A

### CODE REVIEW

File: components/next_steps_view/next_steps_tips.tsx [hunk 1]
```diff
@@ -314,28 +314,61 @@ export default function NextStepsTips(props: Props) {
...
+                    <h3>
+                        <FormattedMessage
+                            id='next_steps_view.tips.createChannels'
+                            defaultMessage='Create a new channel'
+                        />
+                    </h3>
+                    <FormattedMessage
+                        id='next_steps_view.tips.createChannels.text'
+                        defaultMessage={
+                            "Think of a topic you' like to organize a conversation around."
```
Comment (devinbinnie) [inline components/next_steps_view/next_steps_tips.tsx:331]: Typo: should be `you'd`

**Review by devinbinnie — APPROVED** [review]
LGTM! Thanks @kaakaa!
Just one non-blocking comment.

File: components/next_steps_view/next_steps_tips.tsx (hunk 1 above)
Comment (kaakaa) [reply components/next_steps_view/next_steps_tips.tsx]: Thanks @devinbinnie  👍

**Review by nickmisasi — APPROVED** [review]
LGTM. Thanks @kaakaa

**Review by stevemudie — APPROVED** [review]
LGTM

**Pull request merged**